tools/reviewer/web/src/app/dashboard/reviews-panel/review/review-titlebar/review-titlebar.component.html

```diff
@@ -1,93 +1,42 @@
-<div class="review-title" fxFlex>
+<div class="review-title" fxFlex *ngIf="!isLoading">
   <div fxFlex="270px">
-    <span class="diffNumber"> Diff {{ diff.number }} </span> 
+    <span class="diffNumber"> Diff {{ diff.getId() }} </span>
     by
-    <span [ngClass]="{ 
-      'bold-text': 
-        diff.needAttentionOfList.includes(diff.author),
-        'author-name': true
-      }">
-      <button mat-raised-button 
+    <span
+      [ngClass]="{ 
+        'author-name': true,
+        'bold-text': diff.getAuthor().getNeedsattention()
+      }"
+    ><button mat-raised-button 
         class="add-to-list"
         *ngIf="editable"
-        (click)="saveAttentionList(diff.author);
-        $event.stopPropagation()"
-        >
+        (click)="updateAuthor()"
+      >
         {{ 
-          diff.needAttentionOfList.indexOf(diff.author) > -1
-          ? 'Remove from Needs Attention'
-          : 'Add to Needs Attention'
+          diff.getAuthor().getNeedsattention()
```

Comment:
s/getNeedsattention/getNeedsAttention

Reply:
:)

Reply:
😆  @oferb join the club!

Reply:
:-)
LOL
GMTA

Reply:
@oferb oh, wow. It's a mistake in proto. Camel case instead of snake case.
Cool. Yeah, I'll leave a comment